packages/internal/generated-clients/src/imx/api.ts

```diff
@@ -2,9 +2,9 @@
 /* eslint-disable */
 /**
  * Immutable X API
- * No description provided (generated by Openapi Generator https://github.com/openapitools/openapi-generator)
+ * Immutable X API
  *
- * The version of the OpenAPI document: 3.0
+ * The version of the OpenAPI document: 3.0.0
```

Comment:
what consideration made this happen?

Reply:
The original looks like an error on our part? Versions should have been using semver format from the start? (We do that for all our SDKs)